sklearn/metrics/cluster/tests/test_unsupervised.py

```diff
@@ -282,6 +284,19 @@ def test_silhouette_nonzero_diag(dtype):
         silhouette_samples(dists, labels, metric="precomputed")
 
 
+@pytest.mark.parametrize("to_sparse", (csr_matrix, csc_matrix, dok_matrix, lil_matrix))
+def test_silhouette_samples_sparse(to_sparse):
+    """Check that silhouette_samples works for sparse matrices correctly"""
```

Comment:
I think that we should specified that this is with the precomputed metric.
I think that we should have an additional test without precomputed metric.

Reply:
I have added the additional test with the default Euclidean metric.